omero/sysadmins/unix/install-web-trial.txt

```diff
@@ -0,0 +1,348 @@
+OMERO.web WSGI deployment (TRIAL)
+=================================
+
+OMERO.web is the web application component of the OMERO platform which
+allows for the management, visualization (in a fully multi-dimensional
+image viewer) and annotation of images from a web browser. It also
+includes the ability to manage users and groups.
+
+Deploying OMERO.web separately from OMERO.server is recommended as they
+perform best under different circumstances and require a different set of
+dependencies. This guide uses the example of deploying with nginx and
+gunicorn.
+
+Prerequisites
+-------------
+
+-  Python 2.7
+
+   -  :pypi:`virtualenv` (optional) tool to create isolated Python environments
+
+   -  :pypi:`PyPI <>` a package management system used to install and manage
+      software packages written in Python. PyPI is already installed if
+      you are using Python 2 >=2.7.9
+
+   -  `omego <https://github.com/ome/omego>`_ OME package management system
+
+   -  `ZeroC`_ IcePy 3.6 (IcePy 3.5 is also supported although requires global
+      installation from RPM)
+
+   -  `Pillow`_ <3.4
+
+   -  `NumPy <http://www.numpy.org>`_ >=1.9 
+
+-  A `WSGI <http://wsgi.readthedocs.org>`_-capable web server such as
+   `nginx <http://nginx.org/>`_ and `gunicorn <http://docs.gunicorn.org/>`_,
+
+
+Gunicorn default configuration
+------------------------------
```

Comment:
It seems that the instructions following this heading are more about "Installing requirements" rather than "Gunicorn default configurations"? Would a different heading make more sense?